packages/components/pie-radio/src/defs-react.ts

```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import type React from 'react';
 /**
  * TODO: Verify if ReactBaseType can be set as a more specific React interface
  * Use the React IntrinsicElements interface to find how to map standard HTML elements to existing React Interfaces
```

Comment:
Not sure if you plan to do it later but this comment can be removed and ReactBaseType can be set to `React.InputHTMLAttributes<HTMLInputElement>`